src/Compute/Compute/Generated/DedicatedHost/DedicatedHostGetMethod.cs

```diff
@@ -48,8 +48,8 @@ public override void ExecuteCmdlet()
                 {
                     case "ResourceIdParameter":
                         resourceGroupName = GetResourceGroupName(this.ResourceId);
-                        hostGroupName = GetResourceName(this.ResourceId, "Microsoft.Compute/hosts", "hosts");
-                        hostName = GetResourceName(this.ResourceId, "Microsoft.Compute/hosts", "hosts");
+                        hostGroupName = GetResourceName(this.ResourceId, "Microsoft.Compute/hostGroups", "hosts");
+                        hostName = GetInstanceId(this.ResourceId, "Microsoft.Compute/hostGroups", "hosts");
                         break;
```

Comment:
Add an example to the Get-AzHost.md file for this if it does not exist.

Reply:
It does not exist.  I'll add that too